news/tasks/workflows.py

```
@@ -0,0 +1,253 @@
+            f"성공={crawl_job.successful_articles}, "
+            f"중복={result.get('duplicates_removed', 0)}"
+        )
+
+    except CrawlJob.DoesNotExist:
+        logger.error(f"[Workflow] CrawlJob {job_id}를 찾을 수 없습니다.")
+    except Exception as e:
+        logger.error(f"[Workflow] CrawlJob {job_id} 완료 처리 실패: {str(e)}")
+        try:
+            crawl_job = CrawlJob.objects.get(id=job_id)
+            crawl_job.mark_as_failed(str(e))
+        except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)